### PR TITLE
Update donate@

### PR DESCRIPTION
**Pull request by RhinosF1**
Now handled by Owen.

Will contact Owen & Labster to notify

### CODE REVIEW

**Comment by labster** [comment]
LGTM.  I closed the Paypal account, so there's no need for me to have access to this email now.  You should be able to reopen at the same address; let me know if there's a problem.

File: modules/postfix/files/aliases [hunk 1]
```diff
@@ -47,7 +47,7 @@ rhinosf1: rhinosf1@gmail.com
 campaign: sre
 
 # board
-board: owen,southparkfan,void
+board: owen,southparkfan,void,robla
```
Comment (paladox) [inline modules/postfix/files/aliases]: Seems wrong as Reception added this?

Comment (RhinosF1) [reply modules/postfix/files/aliases]: I rebased it, not sure what the diff is doing

**Comment by RhinosF1** [comment]
13:01:58 <RhinosF1> Leave to SPF|Cloud pls

**Comment by RhinosF1** [comment]
@Southparkfan

**Pull request merged**